src/audit/github_env.rs

```diff
@@ -143,9 +192,21 @@ impl WorkflowAudit for GitHubEnv {
         pwsh_parser
             .set_language(&pwsh)
             .context("failed to load powershell parser")?;
+
+        let bash_redirect_query = Query::new(&bash.into(), BASH_REDIRECT_QUERY)?;
+        let bash_pipeline_query = Query::new(&bash.into(), BASH_PIPELINE_QUERY)?;
+
         Ok(Self {
             bash_parser: RefCell::new(bash_parser),
             pwsh_parser: RefCell::new(pwsh_parser),
+            bash_redirect_query_span_idx: bash_redirect_query
```

Comment:
NB: I cache these capture indices at the struct level because they vary between individual queries, and because extracting them is technically fallible (although infallible in practice, as long as there's no typo). But it's definitely a bit of an eyesore.